theurgy.lic

```diff
@@ -511,10 +498,7 @@ class TheurgyActions
     elsif DRSpells.known_spells.include? 'Bless'
       DRCI.get_item_safe('some oil', @theurgy_supply_container)
       complete_or_interrupt_research
-      DRCA.cast_spell({ 'abbrev' => 'bless',
-                   'mana' => 1,
-                   'prep_time' => 2,
-                   'cast' => 'cast oil' }, @settings)
+      DRCTH.quick_bless_item('oil', @settings)
```

Comment:
Should `DRCTH` also have a quick_gg function? See https://github.com/rpherbig/dr-scripts/blob/64940ff0d16f7246a032fe711b42b83af2219bbc/theurgy.lic#L463

I think that would let us remove `common-arcana` from this file, too.

Reply:
Two thoughts. How about a quick-cast method in DRCA where we can specify the spell? 

and/or (and this is something I planned to add here anyway) we allow for users to define their own quick* waggles, since the main thing I was trying to solve here anyway, is the fact that some clerics can't always quick-cast at 2 seconds even at 1 mana (low concentration, or other reasons) that would allow users to work around our defaults if they aren't suited.